mrtt-ui/src/components/RestorationAimsForm/StakeholderBenefitsInputs.js

```diff
@@ -0,0 +1,95 @@
+import { useWatch } from 'react-hook-form'
```

Comment:
This is a component I made to be given to CheckboxGroup for the nested questions